tests/test_TC_014_02_return_to_catalog.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+# @Time    : 2022/12/12 19:00
+# @Author  : Natalia Ts
+
+import allure
+from selenium.common import NoSuchElementException
+from pages.products_page import ProductsPage
+from pages.login_page import LoginPage
+from pages.cart_page import CartPage
+import pytest
```

Comment:
According to PEP8, imports should always be placed at the top of the file, immediately after any module comments and docstrings